traitcuration/traits/datasources/dummy.py

```diff
@@ -14,14 +14,14 @@ def import_dummy_data():
     Mapping.objects.all().delete()
     Trait.objects.all().delete()
     OntologyTerm.objects.all().delete()
-    User.objects.all().delete()
+    User.objects.exclude(email="admin@admin.com").delete()
     # ONTOLOGY TERMS
     term1 = OntologyTerm(label='/ Diabetes mellitus /', curie='EFO:0000400',
                          iri='http://www.ebi.ac.uk/efo/EFO_0000400', status=Status.CURRENT)
     term2 = OntologyTerm(label='/ digestive system disease /', curie='EFO:0000405',
                          iri='http://www.ebi.ac.uk/efo/EFO_0000405', status=Status.CURRENT)
-    term3 = OntologyTerm(label='/ Insulin-resistant diabetes mellitus /', curie='HP:0000831',
-                         iri='http://purl.obolibrary.org/obo/HP_0000831', status=Status.NEEDS_IMPORT)
+    term3 = OntologyTerm(label='/ Hereditary breast cancer /', curie='Orphanet:227535',
```

Comment:
It would be useful to add a comment here that the term here is purposefully incorrect to test OLS updates

Reply:
Just to clarify, by comment you mean a Python comment or a comment in the term label like *- INCORRECT* ?

Reply:
Well... Originally I meant the Python comment, but now that you said it, adding “INCORRECT” to the label would also be useful :)